extension/js/common/core/crypto/pgp/openpgp-key.ts

```diff
@@ -588,15 +590,15 @@ export class OpenPGPKey {
         opgp.enums.read(opgp.enums.publicKey, opgp.enums.publicKey.eddsa),
       ].includes(key.getAlgorithmInfo().algorithm)
     ) {
-      return 0; // disallow encryption for these algorithms
+      return this.noKeyFlags; // disallow encryption for these algorithms
     }
     return verifiedSignature.keyFlags[0] & (opgp.enums.keyFlags.encryptCommunication | opgp.enums.keyFlags.encryptStorage);
   };
 
   private static getValidSigningKeyPacketFlags = (key: OpenPGP.Key | OpenPGP.Subkey, verifiedSignature: OpenPGP.SignaturePacket): OpenPGP.enums.keyFlags => {
     if (!verifiedSignature.keyFlags || verifiedSignature.revoked !== false) {
       // Sanity check
-      return 0;
+      return this.noKeyFlags;
```

Comment:
@sosnovsky  Not sure if this the best solution.
Maybe you could suggest better way?